tests/test_message/event_sample.py

```diff
@@ -0,0 +1,204 @@
+
+request = {
+  "ref": "refs/heads/main",
+  "before": "034c31e8e46431711bed9eada103aa552aaee6dd",
+  "after": "443f68ce112ab055bbf48666762a626baa4729f6",
+  "repository": {
+    "id": 807808293,
+    "node_id": "R_kgDOMCYtJQ",
+    "name": "demo_repo",
+    "full_name": "brngylni/demo_repo",
+    "private": True,
+    "owner": {
+      "name": "brngylni",
+      "email": "66281850+brngylni@users.noreply.github.com",
+      "login": "brngylni",
+      "id": 66281850,
+      "node_id": "MDQ6VXNlcjY2MjgxODUw",
+      "avatar_url": "https://avatars.githubusercontent.com/u/66281850?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/brngylni",
+      "html_url": "https://github.com/brngylni",
+      "followers_url": "https://api.github.com/users/brngylni/followers",
+      "following_url": "https://api.github.com/users/brngylni/following{/other_user}",
+      "gists_url": "https://api.github.com/users/brngylni/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/brngylni/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/brngylni/subscriptions",
+      "organizations_url": "https://api.github.com/users/brngylni/orgs",
+      "repos_url": "https://api.github.com/users/brngylni/repos",
+      "events_url": "https://api.github.com/users/brngylni/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/brngylni/received_events",
+      "type": "User",
+      "site_admin": False
+    },
+    "html_url": "https://github.com/brngylni/demo_repo",
```

Comment:
Is there a way to emulate this without using a secondary external repository?

I want our test cases to be self-sufficient and not dependent on external factors.

Reply:
Do you mean a seperate repository under fedora infra?

Reply:
@brngylni I mean by not having an external repository at all.

Reply:
I thought having a real request might be helpful in terms of testing the hash and etc. That's why I used a real payload. I can edit the payload if you'd like to.